recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
 
 build:
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
   skip: True  # [win]
```

pypy build is failing on macOS, so just skip
```suggestion
  skip: True  # [win or (osx and python_impl == 'pypy')]
```